sdk/provisioner.go

```diff
@@ -107,6 +107,15 @@ func (out *ProvisionOutput) AddArgs(args ...string) {
 	out.CommandLine = append(out.CommandLine, args...)
 }
 
+// AddArgsImmediatelyAfterExecutableName can be used to add additional arguments to the command line of the provision output, but immediately after the executable name. The rest of the command line appear after the additional arguments.
+func (out *ProvisionOutput) AddArgsImmediatelyAfterExecutableName(args ...string) {
```

Comment:
Can we call this function `PrependArgs` (or `AddArgsFirst`) and rename `AddArgs` to `AppendArgs` (or `AddArgsLast`)?